### DISCUSSION TITLE
v2/zh/config/plugins/intro

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#307] [Announcements]
# v2/zh/config/plugins/intro

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/config/plugins/intro.html

**Comment by VivaAlexYu** [comment]
能否出个mp4文件视频播放插件

**Reply by Mister-Hope** [reply]
PR welcomed as described in https://github.com/vuepress-theme-hope/vuepress-theme-hope/issues/1847

**Comment by VivaAlexYu** [comment]
Please arrange to vuepress-plugin-next-search the plugin

**Reply by Mister-Hope** [reply]
That plugin is pretty bad. I will rebuild a new plugin when I have time.

Also, I am negative towards downloading a search index database and perform searching on the client side.
